Lib/unittest/__init__.py

```diff
@@ -57,7 +57,6 @@ def testMultiply(self):
 __unittest = True
 
 from .result import TestResult
-from .async_case import IsolatedAsyncioTestCase
```

Comment:
Maybe add a comment here to explain that IsolatedAsyncioTestCase is imported lazily?